### PR TITLE
fix naming convention between access log format vs encoding

### PR DESCRIPTION
**Pull request by GregHanson**

### CODE REVIEW

**Comment by GregHanson** [comment]
related to https://github.com/istio/istio/issues/10056

File: mesh/v1alpha1/config.proto [hunk 1]
```diff
@@ -121,10 +121,14 @@ message MeshConfig {
...
   
-  // Format for the proxy access log (text or json).
-  // Default value is text.
+  // Format for the proxy access log 
+  // Empty value results in proxy's default access log format 
   string access_log_format = 24;
 
+  // Encoding for the proxy access log (text or json).
+  // Default value is text.
+  string access_log_encoding = 27;
```
Comment (ozevren) [inline mesh/v1alpha1/config.proto]: Shouldn't this be an enum?

Comment (GregHanson) [reply mesh/v1alpha1/config.proto]: @mandarjog @rshriram care to weigh in?

Comment (mandarjog) [reply mesh/v1alpha1/config.proto]: Makes sense for this to be `enum`. 
The other PR probably is merged by now, so needs the same fix ...

Comment (GregHanson) [reply mesh/v1alpha1/config.proto]: done

**Pull request merged**